### PR TITLE
@maxcbc/further log updates

### PR DESCRIPTION
**Pull request by maxcbc**
add route to controller logs
fix dodgy type check

### CODE REVIEW

File: server/lib/enrich/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = exports = function enrich(content, format) {
 		content.type = content.content_type;
 	}
 
-	if (content instanceof Object && !(content instanceof Error) && content.type in exports) {
+	if (contenta.constructor.name === 'Object' && !(content instanceof Error) && content.type in exports) {
```
Comment (leggsimon) [inline server/lib/enrich/index.js]:
```suggestion
	if (content.constructor.name === 'Object' && !(content instanceof Error) && content.type in exports) {
```

File: server/controllers/backup.js [hunk 2]
```diff
@@ -42,7 +42,10 @@ module.exports = exports = async (req, res, next) => {
 		});
 	}
 	catch(error) {
-		log.error({error});
+		log.error({
+			route: req.route.path,
```
Comment (leggsimon) [inline server/controllers/backup.js]: These aren't easily searchable in the codebase though

Comment (maxcbc) [reply server/controllers/backup.js]: OK will do again

Comment (leggsimon) [reply server/controllers/backup.js]: Ideally, some sort of searchable string would be good, even if it’s something like the function name uppercased

Comment (maxcbc) [reply server/controllers/backup.js]: REQUEST_COMPLETED_MATE

**Pull request merged**